.github/ISSUE_TEMPLATE/feature_tracking.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Tracking Issue
+about: Collect discussion and track the progress of a shiny new feature
+title: ''
+labels: C-Enhancement, C-Tracking-Issue S-Needs-Triage
+assignees: ''
+---
+
+## What does this new feature do?
+
+**NOTE:** please only create tracking issues for features that have a reasonably clear consensus and implementation plan (although no definite timeline is needed).
+For feature requests and early design work, please create a [Discussion](https://github.com/bevyengine/bevy/discussions) instead.
+
+In a short paragraph, describe what the feature does.
+Try to focus on the new functionality it allows, rather than the implementation details.
+
+## What still needs to be done?
+
+- [ ] Add
+
+- [ ] Items
+
+- [ ] Here
+
```

Comment:
I think bulleted lists like this should be "consolidated" (no newlines between them).

- [ ] Add
- [ ] Items
- [ ] Here